<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LocationList</class>
 <widget class="QWidget" name="LocationList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LocationList</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>431</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="cityName_Lbl">
        <property name="text">
         <string>CityName</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="cityName_in"/>
      </item>
      <item>
       <widget class="QPushButton" name="getCityButton">
        <property name="text">
         <string>Get</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="listWidget">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QListView::item {
    padding-top: 10px;
    padding-bottom: 10px;
}</string>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="itemAlignment">
       <set>Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
